meteor/client/ui/RundownView/RundownTiming/RundownTimingProvider.tsx

```diff
@@ -121,6 +121,9 @@ export const RundownTimingProvider = withTracker<
 		private temporaryPartInstances: Map<PartId, PartInstance> = new Map<PartId, PartInstance>()
 
 		private linearParts: Array<[PartId, number | null]> = []
+		private prevPartId: string = ''
```

Comment:
Why is this an empty string? This is only ever compared with PartInstanceIds, so I would expect this should be `private prevPartId = PartInstanceId | null = null`. Please add a comment what this is used for and why.

Reply:
In the first setup i did the string I compared it with could be undefined, so i ended up giving it a value.
But as the first comparison now always will be with a string, | null = null is way better 👍

Reply:
Has been fixed :)